src/probnum/quad/_bayesquad.py

```diff
@@ -31,6 +31,7 @@ def bayesquad(
     measure: Optional[IntegrationMeasure] = None,
     method: str = "vanilla",
     policy: str = "bmc",
+    rng: np.random.Generator = None,
```

Comment:
Why do we want `rng` to be an argument in the first place? Why would I want to use a different `rng` other than the default one?

Reply:
I messaged you offline :)